# Pathology Models and Spatial Transcriptomics Models Articles

## Pathology Models
### Subfield: [Multimodal]
- **CONCH**: A visual-language foundation model for computational pathology, highlighting its performance in tasks such as image classification and segmentation. [[Nature Medicine, 2024](https://www.nature.com/articles/s41591-024-02856-4)]

- **Pathology language–image pretraining (PLIP)**: A visual–language foundation model for pathology image analysis using medical Twitter, which demonstrates state-of-the-art performance in classifying new pathology images. [[Nature Medicine, 2023](https://www.nature.com/articles/s41591-023-02504-3?fromPaywallRec=false)]

- **Pathchat**: A Multimodal Generative AI Copilot for Human Pathology, exploring the integration of AI in pathology to assist human pathologists. [[Nature, 2024](https://www.nature.com/articles/s41586-024-07618-3#citeas)]

- **Medical multimodal large language model (Med-MLLM)**: A medical multimodal large language model for future pandemics, designed to rapidly deploy and adapt to rare diseases with limited labels. [[npj Digital Medicine, 2023](https://www.nature.com/articles/s41746-023-00952-2?fromPaywallRec=false)]

- **Prov-GigaPath**: A whole-slide foundation model for digital pathology from real-world data, demonstrating robust performance in various pathology tasks. [[Nature, 2024](https://www.nature.com/articles/s41586-024-07441-w?fromPaywallRec=false#data-availability)]

- **UNI**: Towards a general-purpose foundation model for computational pathology, evaluated on diverse tasks, advancing unsupervised representation learning at scale in pathology. [[Nature Medicine, 2024](https://www.nature.com/articles/s41591-024-02857-3?fromPaywallRec=false)]

- **TopACT**: Multiscale topology classifies cells in subcellular spatial transcriptomics, improving cell classification and morphology for various tissues. [[Nature, 2024](https://www.nature.com/articles/s41586-024-07563-1?fromPaywallRec=false)]
  
- **LabelEval**: Labelling instructions matter in biomedical image analysis, emphasizing the importance of high-quality annotation guidelines for improving the performance of biomedical image analysis algorithms. [[Nature Machine Intelligence, 2023](https://www.nature.com/articles/s42256-023-00625-5)]
- **Histomorphological Phenotype Learning (HPL)**: Mapping the landscape of histomorphological cancer phenotypes using self-supervised learning on unannotated pathology slides, creating an atlas of histomorphological phenotypes. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-48666-7)]

- **AI-based histopathology image analysis**: AI-based histopathology image analysis reveals a distinct subset of endometrial cancers, identifying a biologically distinct subgroup with inferior progression-free and disease-specific survival. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-49017-2)]

- **AI for cervical cytology**: Artificial intelligence enables precision diagnosis of cervical cytology grades and cervical cancer, showcasing high accuracy and significant improvements in diagnostic performance with the assistance of cytopathologists. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-48705-3)]

-  **Teacher-student collaborated multiple instance learning**: Teacher-student collaborated multiple instance learning for pan-cancer PDL1 expression prediction from histopathology slides, using bulk RNA sequencing and dynamic pseudo-labels for accurate analysis. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-46764-0)]

- **UniFMIR**: Pretraining a foundation model for generalizable fluorescence microscopy-based image restoration, demonstrating improved restoration precision, generalization, and versatility across various microscopy tasks and datasets [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02244-3)]
- **ELD**: Spatial landmark detection and tissue registration with deep learning, introducing a neural-network-guided thin-plate spline method to enhance accuracy and stability in landmark detection and tissue registration across diverse datasets and modalities [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02199-5)]
- **TriPath**: TriPath is a 3D pathology deep learning platform for clinical endpoint prediction, outperforming 2D slice-based approaches and pathologist baselines by using larger tissue volumes to mitigate sampling bias and account for tissue heterogeneity, demonstrating significant potential in clinical applications for recurrence risk-stratification in prostate cancer [[Cell, 2024](https://www.sciencedirect.com/science/article/pii/S0092867424003519)]
- **HeartBEiT**: A foundational vision transformer for ECG analysis, pre-trained on 8.5 million ECGs, outperforming standard CNN architectures in diagnosing hypertrophic cardiomyopathy, low left ventricular ejection fraction, and ST elevation myocardial infarction. HeartBEiT improves performance with smaller datasets and enhances explainability by highlighting biologically relevant regions of the ECG, demonstrating significant potential for clinical applications in ECG interpretation [[npj Digital Medicine, 2023](https://www.nature.com/articles/s41746-023-00840-9)]
- **NaroNet**: Weakly supervised deep learning predicts recurrence in low-grade endometrial cancer from multiplexed immunofluorescence images, outperforming current prognostic factors. The model learns complex tumor-immune interrelations at local phenotypes, cellular neighborhoods, and tissue areas without manual annotation, achieving high predictive accuracy and providing an AI-based spatial phenotyping framework for early-stage cancer prognosis [[npj Digital Medicine, 2023](https://www.nature.com/articles/s41746-023-00795-x)]
- **CardioNet**: Multimodal deep learning model predicts cardiovascular risk by integrating fundus abnormalities and traditional risk factors, achieving high accuracy in internal and external validations. The model uses deep neural networks to analyze clinical risk factors and fundus photographs, significantly improving CVD prediction performance compared to traditional methods, and demonstrating the potential of fundus imaging as a non-invasive predictor for cardiovascular disease [[npj Digital Medicine, 2023](https://www.nature.com/articles/s41746-023-00748-4)]
- **Multimodal Aging Estimator**: A Transformer-based model estimates biological age (BA) using facial, fundus, and tongue images, demonstrating accurate BA predictions. Significant differences in BA versus chronological age (AgeDiff) were found in individuals with chronic diseases, highlighting AgeDiff's potential as a biomarker for detecting and predicting the progression of chronic diseases. This approach uses easily acquired patient data to identify and manage chronic health conditions [[PNAS, 2023](https://www.pnas.org/doi/10.1073/pnas.2308812120)]
- **SynthSeg**: SynthSeg is an image segmentation tool for automated analysis of highly heterogeneous brain MRI scans, using a novel training strategy for deep neural networks that allows robust analysis without retraining. It includes whole-brain segmentation, cortical parcellation, intracranial volume estimation, and automatic detection of faulty segmentations, facilitating use and enhancing reproducibility. SynthSeg is publicly available with FreeSurfer and has the potential to transform neuroimaging studies by unlocking the analysis of vast clinical data sets [[PNAS, 2023](https://www.pnas.org/doi/10.1073/pnas.2216399120)]


## Spatial Transcriptomics Models
### Subfield: [Multimodal]

- **Spatial redundancy denoising transformer (SRDTrans)**: Spatial redundancy transformer for self-supervised fluorescence image denoising, offering state-of-the-art performance in denoising tasks for single-molecule localization microscopy and two-photon volumetric calcium imaging. [[Nature Computational Science, 2023](https://www.nature.com/articles/s43588-023-00568-2)]

- **STAligner**: Integrating spatial transcriptomics data across different conditions, technologies, and developmental stages, enabling spatially aware data integration, spatial domain identification, and comparative analysis. [[Nature Computational Science, 2023](https://www.nature.com/articles/s43588-023-00528-w)]


- **Reconciling privacy and accuracy**: Reconciling privacy and accuracy in AI for medical imaging, addressing the trade-off between robust performance and stringent privacy measures using differential privacy. [[Nature Machine Intelligence, 2024](https://www.nature.com/articles/s42256-024-00858-y)]

- **Foundation model for cancer imaging biomarkers**: Developing a foundation model for cancer imaging biomarker discovery, showing improved learning and stability in clinical applications. [[Nature Machine Intelligence, 2024](https://www.nature.com/articles/s42256-024-00807-9)]

- **STitch3D**: Construction of a 3D whole organism spatial atlas by joint modelling of multiple slices with deep neural networks, integrating multiple ST slices to reconstruct 3D cellular structures. [[Nature Machine Intelligence, 2023](https://www.nature.com/articles/s42256-023-00734-1)]

- **Uncertainty-guided dual-views**: Uncertainty-guided dual-views for semi-supervised volumetric medical image segmentation, leveraging adversarial learning for accurate and robust segmentation. [[Nature Machine Intelligence, 2023](https://www.nature.com/articles/s42256-023-00682-w)]

- **Sopa**: A technology-invariant pipeline for analyses of image-based spatial omics, providing a memory-efficient framework for various spatial omics technologies. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-48981-z)]

- **scCube**: Simulating multiple variability in spatially resolved transcriptomics with scCube, a tool for unbiased simulation of spatial transcriptomics data. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-49445-0)]

- **DOT**: A flexible multi-objective optimization framework for transferring features across single-cell and spatial omics, integrating data modalities to improve accuracy. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-48868-z)]

- **TRACERx-PHLEX**: Deep cell phenotyping and spatial analysis of multiplexed imaging with TRACERx-PHLEX, combining deep learning-based segmentation, automated cell-type annotation, and spatial analysis. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-48870-5)]

- **stKeep**: Dissecting tumor microenvironment from spatially resolved transcriptomics data by heterogeneous graph learning, identifying cell-modules and gene-modules, and inferring cell-cell communication patterns. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-49171-7)]


- **Pianno**: A probabilistic framework automating semantic annotation for spatial transcriptomics, enabling precise annotation of diverse anatomical structures and tumor microenvironments. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-47152-4)]


- **BIDCell**: Biologically-informed self-supervised learning for segmentation of subcellular spatial transcriptomics data, integrating cell-type data and cell morphology information to enhance segmentation accuracy. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-023-44560-w)]

- **PROST**: Quantitative identification of spatially variable genes and domain detection in spatial transcriptomics using the PROST framework, integrating spatial and transcriptional data for superior SVG identification and domain segmentation. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-024-44835-w)]

- **MENDER**: Fast and scalable tissue structure identification in spatial omics data, offering substantial improvements in accuracy, scalability, and running time over existing complex models. [[Nature Communications, 2024](https://www.nature.com/articles/s41467-023-44367-9)]

- **The covariance environment (COVET)**: The covariance environment defines cellular niches for spatial inference, utilizing gene-gene covariance and optimal transport-based metrics for robust spatial transcriptomic analysis. [[Nature Biotechnology, 2024](https://www.nature.com/articles/s41587-024-02193-4)]

- **Starfysh**: Starfysh integrates spatial transcriptomic and histologic data to reveal heterogeneous tumor-immune hubs, using a deep generative model to improve the characterization of spatial dynamics in complex tissues. [[Nature Biotechnology, 2024](https://www.nature.com/articles/s41587-024-02173-8)]

- **iStar**: Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology, enhancing gene expression resolution to near-single-cell levels and enabling gene expression prediction in histology-only sections. [[Nature Biotechnology, 2024](https://www.nature.com/articles/s41587-023-02019-9)]

- **MaxFuse**: Integration of spatial and single-cell data across modalities with weakly linked features, achieving high-quality integration through iterative coembedding, data smoothing, and cell matching [[Nature Biotechnology, 2023](https://www.nature.com/articles/s41587-023-01935-0)]

- **scDesign3**: scDesign3 generates realistic in silico data for multimodal single-cell and spatial omics, enabling advanced analysis and integration of omics data [[Nature Biotechnology, 2023](https://www.nature.com/articles/s41587-023-01772-1)]

- **GPSA**: Alignment of spatial genomics data using deep Gaussian processes, providing a robust method for aligning distorted spatial samples across different technologies and scales, enabling comprehensive spatially aware analyses [[Nature Methods, 2023](https://www.nature.com/articles/s41592-023-01972-2)]
  
- **IST**: In silico tissue generation and power analysis for spatial omics, providing a framework for generating parameterized tissue models and conducting power analysis for spatially resolved omics data [[Nature Methods, 2023](https://www.nature.com/articles/s41592-023-01766-6)]
- **TISSUE**: Uncertainty-calibrated prediction of single-cell spatial transcriptomics improves downstream analyses, enhancing differential gene expression analysis, clustering, visualization, and supervised learning models by providing well-calibrated prediction intervals for spatial gene expression [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02184-y)]
- **CytoCommunity**: Unsupervised and supervised discovery of tissue cellular neighborhoods from cell phenotypes, using graph neural networks to identify condition-specific and predictive tissue cellular neighborhoods (TCNs) without intermediate clustering, revealing new granulocyte-enriched and cancer-associated fibroblast-enriched TCNs in high-risk tumors [[Nature Methods, 2024](https://www.nature.com/articles/s41592-023-02124-2)]
- **IRIS**: Accurate and efficient integrative reference-informed spatial domain detection for spatial transcriptomics, leveraging single-cell RNA sequencing data to enhance tissue spatial organization characterization, with substantial accuracy and speed improvements across diverse technologies and datasets [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02284-9)]
- **spaVAE**: Dependency-aware deep generative models for multitasking analysis of spatial omics data, providing probabilistic characterization and capturing spatial correlations for enhanced tasks like clustering, dimensionality reduction, and resolution enhancement in spatial transcriptomics, ATAC-seq, and multi-omics data [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02257-y)]
- **STitch3D**: Construction of a 3D whole organism spatial atlas by joint modelling of multiple slices with deep neural networks, providing comprehensive 3D cellular structures from spatial transcriptomics slices, enhancing the understanding of biological processes in 3D space [[Nature Machine Intelligence, 2023](https://www.nature.com/articles/s42256-023-00734-1)]
- **SPIRAL**: Integrating and aligning spatially resolved transcriptomics data across different experiments, conditions, and technologies, enabling comprehensive spatial transcriptome atlas construction with improved batch effect removal and coordinate alignment through graph domain adaptation and optimal transport [[Genome Biology, 2023](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03078-6)]
- **SpatialScope**: Integrating spatial and single-cell transcriptomics data using deep generative models, providing single-cell resolution for spatial transcriptomics, enhancing cell type identification, and gene expression inference, facilitating advanced analysis such as ligand-receptor interactions and spatially differentially expressed genes [[Nature Communications, 2023](https://www.nature.com/articles/s41467-023-43629-w)]
- **COVET-ENVI**: The covariance environment defines cellular niches for spatial inference, introducing the covariance environment (COVET) and environmental variational inference (ENVI) to capture multivariate cellular interactions, enabling high-accuracy spatial context imputation and single-cell data projection through optimal transport-based distance metrics and conditional variational autoencoders [[Nature Biotechnology, 2024](https://www.nature.com/articles/s41587-024-02193-4)]
- **Nicheformer**: A transformer-based foundation model integrating human and mouse dissociated single-cell and spatial transcriptomics data, pretrained on over 110 million cells. Nicheformer excels in spatially-relevant tasks such as spatial density and niche label prediction, bridging dissociated cells with their spatial context, and advancing the field of spatial single-cell analysis through its large-scale resource and novel spatial learning tasks [[bioRxiv, 2024](https://www.biorxiv.org/content/10.1101/2024.04.15.589472v1)]
- **SLAT**: Spatial-linked alignment tool (SLAT) for aligning heterogenous slices, employing a graph-based algorithm with adversarial matching for efficient alignment of spatially resolved omics data, enhancing cell-typing resolution, integrating multiple modalities, and mapping fine-scale spatial-temporal changes [[Nature Communications, 2023](https://www.nature.com/articles/s41467-023-43105-5)]
- **SpatialSSL**: Whole-Brain Spatial Transcriptomics in the Mouse Brain with Self-Supervised Learning, improving cell type classification by integrating molecular expression and spatial location, demonstrating significant performance gains on a large-scale mouse brain atlas with 550 genes and 4,334,174 cells, particularly in class imbalance scenarios [[NeurIPS 2023](https://openreview.net/forum?id=bfD3Fzy7Mb)]
- **SpiceMix**: SpiceMix enables integrative single-cell spatial modeling of cell identity, utilizing probabilistic latent variable modeling to enhance inference of cell types and spatial patterns. It is applied to spatial transcriptome data from brain regions in human and mouse, improving cell identity inference, revealing interpretable spatial metagenes, and uncovering differentiation trajectories [[Nature Genetics, 2023](https://www.nature.com/articles/s41588-022-01256-z)]

### Subfield: [Integration scRNA-seq]
### Subfield: [Enhance]
- **SPICEMIX**: Integrative spatial multi-omics analysis at single-cell resolution using a probabilistic model, allowing joint analysis of transcriptomics, epigenomics, and other omics data to reveal cell-type-specific regulatory programs and spatial domains in tissue sections [[Nucleic Acids Research, 2020](https://academic.oup.com/nar/article/48/18/e107/5909530)]

### Subfield: [Cell segmentation]
- **SCS**: Cell segmentation for high-resolution spatial transcriptomics, combining imaging and sequencing data to improve cell segmentation accuracy and provide subcellular resolution analysis [[Nature Methods, 2023](https://www.nature.com/articles/s41592-023-01939-3)]
- **multimodality cell segmentation**: The multimodality cell segmentation challenge: toward universal solutions, presenting a benchmark with over 1,500 labeled images from diverse experiments and a Transformer-based deep-learning algorithm for versatile microscopy image analysis [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02233-6)]


### Subfield: [Database]
- **SODB**: SODB facilitates comprehensive exploration of spatial omics data, offering a web-based platform with rich data resources and interactive analytical modules for efficient and comprehensive analysis of spatial omics datasets [[Nature Methods, 2023](https://www.nature.com/articles/s41592-023-01773-7)]
- **SpatialData**: An open and universal data framework for spatial omics, enabling integration and analysis of diverse spatial omics datasets through a unified, extensible file format and interactive tools for spatial annotation and multimodal data alignment [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02212-x)]
### Subfield: [Benchmark]
- **benchmarking spatial clustering**: Benchmarking spatial clustering methods with spatially resolved transcriptomics data, providing a comprehensive evaluation of 13 computational methods on various datasets to assess accuracy, spatial continuity, and robustness, offering guidance for method selection and identifying challenges in handling large-scale tasks [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02215-8)]

## Single Cell Models
### Subfield: [Multimodal]
- **contrastiveVI**: Isolating salient variations of interest in single-cell data with contrastiveVI, offering a method to separate shared and treatment-specific variations in single-cell RNA sequencing data for enhanced analysis and interpretation [[Nature Methods, 2023](https://www.nature.com/articles/s41592-023-01955-3)]
- **scPROTEIN**: A versatile deep graph contrastive learning framework for single-cell proteomics embedding, providing robust peptide uncertainty estimation, batch effect removal, and efficient cell clustering, annotation, and spatial proteomic analysis [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02214-9)]
- **scGPT**: Toward building a foundation model for single-cell multi-omics using generative AI, leveraging generative pretrained transformers to achieve superior performance in cell type annotation, multi-batch integration, perturbation response prediction, and gene network inference across a large repository of single-cell data [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02201-0)]
- **scFoundation**: Large-scale foundation model on single-cell transcriptomics, leveraging 100 million parameters to achieve state-of-the-art performance in single-cell analysis tasks such as gene expression enhancement, drug response prediction, and cell type annotation, by pretraining on over 50 million human single-cell transcriptomic profiles [[Nature Methods, 2024](https://www.nature.com/articles/s41592-024-02305-7)]
- **ForceNet**: Neural networks predict traction forces from a single focal adhesion protein field, generalizing to unseen cells, cell types, and biochemical perturbations. Both physics-constrained and agnostic methods learn interpretable rules, revealing cellular forces encoded across two length scales relating to focal adhesions and cell shape, providing a systematic strategy to infer cellular mechanical properties from molecular components [[Cell, 2023](https://www.sciencedirect.com/science/article/pii/S0092867423013314)]
